tests/mdns/main.cpp

```diff
@@ -80,44 +84,52 @@ int Mainloop(Mdns::Publisher &aPublisher)
 void PublishSingleService(void *aContext, Mdns::State aState)
 {
     assert(aContext == &sContext);
+    uint8_t xpanid[XPANID_SIZE] = {0x41, 0x42, 0x43, 0x44, 0x45, 0x46, 0x47, 0x48};
```

Comment:
kSizeExtPanId is defined in common/types.hpp
```suggestion
    uint8_t xpanid[kSizeExtPanId] = {0x41, 0x42, 0x43, 0x44, 0x45, 0x46, 0x47, 0x48};
```